src/lib/hooks/use-quote-result/index.ts

```diff
@@ -101,6 +101,7 @@ export function useQuoteResult(request: QuoteRequest) {
         outputToken,
         inputTokenAmount: inputTokenAmount.toString(),
         publicClient,
+        slippage,
```

Comment:
any other fields we can think of adding here?

Reply:
this is a list of dependencies for the quote to be refetched, I wouldn't do it just for the sake of it. Atm I cannot but if the need arises, then it doesn't cost anything